pkg/platform/controller/cluster/deletion/cluster_deleter.go

```diff
@@ -153,6 +153,17 @@ func (d *clusterDeleter) deleteCluster(ctx context.Context, cluster *platformv1.
 	if err != nil && !errors.IsNotFound(err) {
 		return err
 	}
+
```

Comment:
This process is not a full gc solution, since credential deletion may be missed, the credential garbage may not be deleted.